### PR TITLE
Add support for checking file type before uploading

### PR DESCRIPTION
**Pull request by ltatarev**
Added `checkImageType` & `acceptType` props to validate image file before it's uploaded.

Sorted prop types as much as I could manually, since eslint doesn't seem to work properly.

### CODE REVIEW

**Review by tomislav-arambasic — APPROVED** [review]
LGTM

File: src/components/image-uploader/ImageUploader.jsx [hunk 1]
```diff
@@ -144,10 +155,18 @@ export default class ImageUploader extends React.Component {
   }
 
   handleDrop(files) {
+    const { acceptType, checkImageType } = this.props;
```
Comment (sstimac) [inline src/components/image-uploader/ImageUploader.jsx]: do you need two props here ? you could just use acceptType with the default of any, and run the typecheck every time you upload

**Review by sstimac — APPROVED** [review]
noice. Only one comment, left up for you to decide

File: src/components/image-uploader/ImageUploader.jsx (hunk 1 above)
Comment (ltatarev) [reply src/components/image-uploader/ImageUploader.jsx]: yeah, could omit `checkImageType` from props, with added variable similar to this to properly cover `image/*` type case

**Pull request merged**